src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,39 @@
+package duke.ui;
+
+import duke.commands.Task;
+import duke.data.TaskList;
+
+public class Ui {
+    TaskList tasks;
+
+    public Ui(TaskList tasks) {
+        this.tasks = tasks;
+    }
+    public void greeting() {
```

Comment:
Should this be renamed to printGreeting() so that it more accurately describes the function, since the function prints the greeting to the screen? Renaming it this way would make it sound more like a verb as well.

The same goes for the other methods below in this class.